bc_obps/registration/api/user_operator.py

```diff
@@ -328,29 +376,49 @@ def create_user_operator_contact(request, payload: UserOperatorContactIn):
 ##### PUT #####
 
 
-@router.put("/select-operator/user-operator/{user_id}/update-status")
-def update_user_operator_status(request, user_id: str):
+@router.put("/select-operator/user-operator/{user_guid}/update-status")
+def update_user_operator_user_status(request, user_guid: str):
     raise_401_if_role_not_authorized(request, ["cas_admin", "cas_analyst", "industry_user_admin"])
     current_admin_user: User = request.current_user
+    # need to convert request.body (a bytes object) to a string, and convert the string to a JSON object
     payload = json.loads(request.body.decode())
     status = getattr(UserOperator.Statuses, payload.get("status").upper())
-    user_operator = get_object_or_404(UserOperator, user=user_id)
+    user_operator = get_object_or_404(UserOperator, user=user_guid)
     user_operator.status = status
-
     if user_operator.status in [UserOperator.Statuses.APPROVED, UserOperator.Statuses.REJECTED]:
         user_operator.verified_at = datetime.now(pytz.utc)
         user_operator.verified_by_id = current_admin_user.user_guid
     if user_operator.status in [UserOperator.Statuses.PENDING]:
         user_operator.verified_at = None
         user_operator.verified_by_id = None
-
     data = serializers.serialize(
         "json",
         [
             user_operator,
         ],
     )
+    user_operator.save()
+    return data
+
 
+@router.put("/select-operator/user-operator/operator/{user_operator_id}/update-status")
+def update_user_operator_status(request, user_operator_id: str):
+    raise_401_if_role_not_authorized(request, ["cas_admin", "cas_analyst", "industry_user_admin"])
+    current_admin_user: User = request.current_user
+    # need to convert request.body (a bytes object) to a string, and convert the string to a JSON object
+    payload = json.loads(request.body.decode())
+    status = getattr(UserOperator.Statuses, payload.get("status").upper())
+    user_operator = get_object_or_404(UserOperator, id=user_operator_id)
+    user_operator.status = status
+    if user_operator.status in [UserOperator.Statuses.APPROVED, UserOperator.Statuses.REJECTED]:
+        user_operator.verified_at = datetime.now(pytz.utc)
+        user_operator.verified_by = current_admin_user
+    data = serializers.serialize(
```

Comment:
tech debt card to do this with django ninja/pydantic instead for data processing

Reply:
https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/479